modules/parent-join/src/main/java/org/elasticsearch/join/mapper/ParentJoinFieldMapper.java

```diff
@@ -189,7 +160,6 @@ public String typeName() {
 
         @Override
         public IndexFieldData.Builder fielddataBuilder(String fullyQualifiedIndexName, Supplier<SearchLookup> searchLookup) {
-            failIfNoDocValues();
```

Comment:
can you remind me why this goes away?

Reply:
you can't disable docvalues on the field mapper, so this never fails